src/engine/source/builder/builders/opBuilderHelperFilter.cpp

```diff
@@ -791,4 +792,430 @@ base::Expression opBuilderHelperContainsString(const std::any& definition)
         });
 }
 
+//*************************************************
+//*                Type filters                   *
+//*************************************************
+
+// field: +is_number
+base::Expression opBuilderHelperIsNumber(const std::any& definition)
+{
+    auto [targetField, name, rawParameters] = helper::base::extractDefinition(definition);
+    auto parameters {helper::base::processParameters(rawParameters)};
+    helper::base::checkParametersSize(parameters, 0);
+    name = helper::base::formatHelperFilterName(name, targetField, parameters);
+
+    // Tracing
+    const auto successTrace {fmt::format("[{}] -> Success", name)};
+
+    const auto failureTrace {fmt::format("[{}] -> Failure", name)};
+
+    // Return result
+    return base::Term<base::EngineOp>::create(
+        name,
+        [=, targetField = std::move(targetField)](
+            base::Event event) -> base::result::Result<base::Event>
+        {
+            if (event->exists(targetField) && event->isNumber(targetField))
+            {
+                return base::result::makeSuccess(event, successTrace);
+            }
+            else
+            {
+                return base::result::makeFailure(event, failureTrace);
+            }
+        });
+}
```

Comment:
We should give a concise error trace of why it failed:

```c++
if (!event->exists(targetField)
{ // Error, target field not found
...
}
```
This applies to all added helpers